ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -262,6 +274,34 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-upi
+- as: azure-ipi-fullyprivate-proxy-f28
```

Comment:
Because we did not see any azure e2e ci jobs in 4.11 running in arm ci, we are not supposed to run y-stream upgrade ci jobs here, either.

Reply:
yeah, the same case, I just added the comments on this line.